homeassistant/components/iss/binary_sensor.py

```diff
@@ -46,22 +49,40 @@ def setup_platform(
     add_entities: AddEntitiesCallback,
     discovery_info: DiscoveryInfoType | None = None,
 ) -> None:
-    """Set up the ISS binary sensor."""
-    if None in (hass.config.latitude, hass.config.longitude):
-        _LOGGER.error("Latitude or longitude not set in Home Assistant config")
-        return
+    """Import ISS configuration from yaml."""
+    _LOGGER.warning(
+        "Configuration of the iss platform in YAML is deprecated and will be "
+        "removed in Home Assistant 2022.4; Your existing configuration "
```

Comment:
```suggestion
        "removed in Home Assistant 2022.6; Your existing configuration "
```
Can we change this to give a little bit more time?

Reply:
Yes, I just forgot to bump it.